Tests fail when installing oldest supported dependencies

See https://github.com/zarr-developers/VirtualiZarr/actions/runs/23123309843/job/67161348972

```
FAILED virtualizarr/tests/test_writers/test_icechunk.py::TestRegion::test_write_region_writes_whole_unspecified_dim - AssertionError: Left and right DataArray objects are not close
Differing values:
L
    array([[False,  True, False],
           [False,  True, False],
           [False,  True, False],
           [False,  True, False],
           [False,  True, False],
           [False,  True, False],
           [False,  True, False],
           [False,  True, False],
           [False,  True, False],
           [False,  True, False],
           [False,  True, False],
           [False,  True, False],
           [False,  True, False],
           [False,  True, False],
           [False,  True, False],
           [False,  True, False],
           [False,  True, False],
           [False,  True, False]])
R
    array([[False, False, False, False, False, False, False, False, False, False,
            False, False, False, False, False, False, False, False],
           [ True,  True,  True,  True,  True,  True,  True,  True,  True,  True,
             True,  True,  True,  True,  True,  True,  True,  True],
           [False, False, False, False, False, False, False, False, False, False,
            False, False, False, False, False, False, False, False]])
FAILED virtualizarr/tests/test_parallel.py::test_lithops_executor_with_partial - Failed: Timeout (>300.0s) from pytest-timeout.
```

@TomNicholas do you know if there was anything in https://github.com/zarr-developers/VirtualiZarr/pull/873 that requires newer versions?

--- a/virtualizarr/tests/test_writers/test_icechunk.py
+++ b/virtualizarr/tests/test_writers/test_icechunk.py
@@ -845,11 +845,9 @@ class TestRegion:
                 zarr_format=3,
             ) as ds,
         ):
-            is_written = ds["y"].isin(my_vds["y"]).broadcast_like(ds)
-            xrt.assert_allclose(
-                is_written,
-                ds["foo"] != 0,
-            )
+            actual = ds["foo"] != 0
+            expected = ds["y"].isin(my_vds["y"]).broadcast_like(actual).transpose(*actual.dims)
+            xrt.assert_allclose(expected, actual)
 
     def test_write_region_auto_without_dimension(
         self,

--- a/virtualizarr/tests/test_writers/test_icechunk.py
+++ b/virtualizarr/tests/test_writers/test_icechunk.py
@@ -846,7 +846,9 @@ class TestRegion:
             ) as ds,
         ):
             actual = ds["foo"] != 0
-            expected = ds["y"].isin(my_vds["y"]).broadcast_like(actual).transpose(*actual.dims)
+            expected = (
+                ds["y"].isin(my_vds["y"]).broadcast_like(actual).transpose(*actual.dims)
+            )
             xrt.assert_allclose(expected, actual)
 
     def test_write_region_auto_without_dimension(

